qiime/normalize_table.py

```diff
@@ -42,20 +48,22 @@ def multiple_file_normalize_CSS(input_dir, output_dir, output_CSS_statistics):
 
     for fname in file_names:
         base_fname, ext = splitext(fname)
-        print base_fname
         original_fname = base_fname+'.biom'
-        json_fname = base_fname+'_json.biom'
         hdf5_infile = join(input_dir, original_fname)
-        json_infile = join(input_dir, json_fname)
-        open(str(json_infile),'w').write(load_table(hdf5_infile).to_json('forR'))
+        tmp_bt = load_table(hdf5_infile) 
         outfile = join(output_dir, 'CSS_'+base_fname+'.biom')
         if output_CSS_statistics:
             output_CSS_statistics = join(output_dir, 'CSS_statistics_'+base_fname+'.txt')
-        run_CSS(json_infile, outfile, output_CSS_statistics=output_CSS_statistics)
-        remove(json_infile)
+
+        with tempfile.NamedTemporaryFile(dir=get_qiime_temp_dir(),
+                                     prefix='QIIME-differential-abundance-temp-table-',
```

Comment:
fix indentation on these two lines (and in the other functions where this occurs)